--- ./src/platform/framehandlers/bpf/bpflib.c.org	2012-03-11 22:21:45.215595328 +0100
+++ ./src/platform/framehandlers/bpf/bpflib.c	2012-03-11 22:23:00.465700901 +0100
@@ -216,7 +216,7 @@ bpf_filter_receive_none(int fd)
 
     prog.bf_len = sizeof(insns) / sizeof(struct bpf_insn);
     prog.bf_insns = insns;
-    return ioctl(fd, BIOCSETF, (u_int)&prog);
+    return ioctl(fd, BIOCSETF, &prog);
 }
 
 /*int
@@ -235,7 +235,7 @@ bpf_filter_receive_none(int fd)
 
   prog.bf_len = sizeof(insns) / sizeof(struct bpf_insn);
   prog.bf_insns = insns;
-  return ioctl(fd, BIOCSETF, (u_int)&prog);
+  return ioctl(fd, BIOCSETF, &prog);
   }*/
 
 int
